svc/jobimp/jobimp.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 The Drivers Cooperative. All rights reserved.
+// Use of this source code is governed by a dual
+// license that can be found in the LICENSE file.
+
+package jobimp
```

Comment:
i would also got rid of "imp" suffix and used just "job" in package name, and everywhere .